examples/nas/spos/readme.md

```diff
@@ -0,0 +1,84 @@
+# Single Path One-Shot Neural Architecture Search with Uniform Sampling
+
+Single Path One-Shot by Megvii Research. [Paper link](https://arxiv.org/abs/1904.00420). [Official repo](https://github.com/megvii-model/SinglePathOneShot).
+
+Block search only. Channel search is not supported yet.
+
+Only GPU version is provided here.
+
+TODO: Reproduction results.
```

Comment:
Can you report the result for now?

Reply:
The result is not aligned with the paper. So no.

Reply:
We need to know the gap, or we cannot merge the branch in case there are bugs.